### ISSUE TITLE
Support logprob in VLM api server[Feature] 

### ISSUE THREAD

**Issue by cjfcsjt** [InternLM/lmdeploy#2041]
### Motivation

I found that the input token logprob is supported by Offline Inference Pipeline, as mentioned in [doc](https://lmdeploy.readthedocs.io/en/latest/inference/vl_pipeline.html#calculate-logits). 
However, when I deployed a VLM server, I found that this feature was not supported. How to enable this on the server? Thanks.

### Related resources

The code I deploy a server is `lmdeploy serve api_server openbmb/MiniCPM-Llama3-V-2_5 --server-port 23333 --session-len 4096`

**Comment by cjfcsjt** [comment]
Anyone can help?
